.gitignore

```diff
@@ -17,3 +17,4 @@ package-lock.json
 lcov*
 coverage/
 .history/*.*
+xtemp/
```

Comment:
what's xtemp?

Reply:
just a folder where you can put some files which aren't tracked via git.

e.g.:
* sometimes I just copy my "real expenses" to my IDE to have a backup if I am testing in Noteplan.
* or I had to rewrite all `2021;02` dates to `2021-02` and I did that in my IDE

Reply:
feels like everyone else doesn't need this though. can you do this locally and discard the change from the PR?

Reply:
ok :D